_members/akul-singhal.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: members
+title: Akul Singhal
+---
+
+I am a 2018 batch undergrad, pursuing Computer Science. I love taking part in competitive coding competitions and I am also interested in software development.
+
+Skills
```

Comment:
Change the `Skills ____` to `## Skills` and likewise for other subheadings.

Reply:
Practice School-1